From 67dcb6ea6a158ac8e058359bc735ab148b134142 Mon Sep 17 00:00:00 2001
From: Laur59 <148864407+Laur59@users.noreply.github.com>
Date: Sat, 8 Feb 2025 08:52:06 +0200
Subject: [PATCH 1/2] =?UTF-8?q?R=C3=A9cup=C3=A8re=20configuration=20global?=
 =?UTF-8?q?e=20depuis=20la=20branche=20arm-software-lva?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

(cherry picked from commit d84ab62b46c3302c556355069c4e92db0240c8cf)
(cherry picked from commit 9f24451bad0006f102dcc46bbec3a3d7173b7746)
---
 arm-software/embedded/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arm-software/embedded/CMakeLists.txt b/arm-software/embedded/CMakeLists.txt
index b47aebf22786..48d266bb530e 100644
--- a/arm-software/embedded/CMakeLists.txt
+++ b/arm-software/embedded/CMakeLists.txt
@@ -151,7 +151,7 @@ set(PARALLEL_LIB_BUILD_LEVELS
 option(
     ENABLE_QEMU_TESTING
     "Enable tests that use QEMU. This option is ON by default."
-    ON
+    OFF
 )
 option(
     ENABLE_FVP_TESTING
@@ -222,6 +222,7 @@ set(LLVM_DISTRIBUTION_COMPONENTS
     llvm-strip
     llvm-symbolizer
     LTO
+    scan-build
     CACHE STRING ""
 )
 set(LLVM_TOOLCHAIN_DISTRIBUTION_COMPONENTS
-- 
2.39.5 (Apple Git-154)

